### DISCUSSION TITLE
PHP executablePath in Windows 11 with WSL2 for VisualStudio Code

### DISCUSSION THREAD

**Discussion by Copylefter** [ddev/ddev#4574] [Q&A]
I'm couldn't use Xdebug tool with VScode on windows machine as it explains in official tutorial https://ddev.readthedocs.io/en/stable/users/debugging-profiling/step-debugging/

VScode asked me for PHP executablePath and I could get in from the docker container.

`Cannot validate since a PHP installation could not be found. Use the setting 'php.validate.executablePath' to configure the PHP executable.`

How it could be resolved?

**Comment by rfay** [comment]
I don't think you should need to do this at all if you just follow the instructions in https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/#visual-studio-code-vs-code-debugging-setup - unless you're doing something other than just step-debugging.

For exotic configurations I think people have convinced vscode to behave by making a wrapper binary, /usr/local/bin/php with contents
```
#!/bin/bash
ddev php $*
```
